sdks/python/apache_beam/io/textio.py

```diff
@@ -83,6 +83,13 @@ def __init__(self, file_pattern, min_bundle_size,
     self._coder = coder
     self._buffer_size = buffer_size
 
+  def display_data(self):
```

Comment:
What do we do in other places? Do we normally include everything or try to come up with a selected set? If the answer is not everything, I think we do not need to display neither buffer_size nor coder.